workspaces/docs/src/components/ArgDescriptor/styles.module.css

```diff
@@ -18,4 +19,14 @@
 .arg-description--required {
   font-size: 14px;
   color: var(--secondary-text-color);
-}
\ No newline at end of file
+}
+
+.arg-description--type {
+  margin-bottom: 10px;
+}
+
+.description {
```

Comment:
Can we update the class name on this component to `.arg-description--description` 

Its verbose I know, but it helps things remain consistent 🙂

Reply:
Yeah sure.